web/tests/hotkey.test.js

```diff
@@ -365,8 +365,6 @@ run_test("misc", ({override}) => {
     assert_mapping("J", navigate, "page_down");
     assert_mapping("k", navigate, "up");
     assert_mapping("K", navigate, "page_up");
-    assert_mapping("s", narrow, "by_recipient");
-    assert_mapping("S", narrow, "by_topic");
```

Comment:
Hmm, shouldn't we still be asserting a mapping for `s`?

Reply:
Sorry, I was earlier under the wrong impression that we were only asserting mappings for hotkeys that called a function unconditionally... Have added a mapping for `s` now!